### PR TITLE
Fixes #18180: Make rtf handle network hostonly uniqueness

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/18180

### CODE REVIEW

File: lib/platform.py [hunk 1]
```diff
@@ -178,14 +178,39 @@ def setup(self, client_path, fail_exit=False):
...
           # update promises on the whole chain
           relays.append(server)
           relays.reverse()
           for h in relays:
             h.run("rudder agent run -u", quiet=False, live_output=True, fail_exit=fail_exit)
           host.run("rudder agent run -ui", quiet=False, live_output=True, fail_exit=fail_exit)
 
+  def pf_id_to_network(self, pf_id):
+    # Since the mask is always 255.255.255.0 this is reasonable
+    return ("192.168.%s.0"%(pf_id), "255.255.255.0")
```
Comment (peckpeck) [inline lib/platform.py:191]: the base network can be defines in the Vagrantfile, so you shouldn't hardcode 192.168. here an the mask is also in the Vagrantfile (see $NETWORK)

**Review by peckpeck — CHANGES_REQUESTED** [review]
Use $NERWORK

**Comment by Fdall** [comment]
PR updated with a new commit

File: lib/platform.py [hunk 2]
```diff
@@ -178,14 +180,50 @@ def setup(self, client_path, fail_exit=False):
...
           for h in relays:
             h.run("rudder agent run -u", quiet=False, live_output=True, fail_exit=fail_exit)
           host.run("rudder agent run -ui", quiet=False, live_output=True, fail_exit=fail_exit)
 
+  def pf_id_to_network(self, pf_id):
+    # Since the mask is always 255.255.255.0 this is reasonable
+    base_subnet = "192.168.0.0/24"
+    with open("Vagrantfile", "r+") as fd:
+      for line in fd:
+        m = re.match("^\$NETWORK=[\'\"](.*)[\'\"]", line)
```
Comment (peckpeck) [inline lib/platform.py:196]: \s* are needed around =

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Review by peckpeck — APPROVED** [review]

**Review by peckpeck — APPROVED** [review]

**Pull request merged**